src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -302,4 +307,14 @@ public String findStatus(UUID resultUuid) {
             return null;
         }
     }
+
+    private Pageable addDefaultSort(Pageable pageable, String defaultSortColumn) {
+        if (pageable.isPaged() && pageable.getSort().getOrderFor(defaultSortColumn) == null) {
+            //if it's already sorted by our defaultColumn we don't add another sort by the same column
+            Sort finalSort = pageable.getSort().and(Sort.by(new Sort.Order(DEFAULT_SORT_DIRECTION, defaultSortColumn)));
```

Comment:
Could be shorten a bit
```suggestion
            Sort finalSort = pageable.getSort().and(Sort.by(DEFAULT_SORT_DIRECTION, defaultSortColumn));
```